helsingborg-stad/app-mitt-helsingborg
# Feat/remove case button

## Explain the changes you’ve made
Added the possibility to delete a case whenever its status type is `active:signature:pending`.
Also refactored calculations to its own component.

## Explain why these changes are made
Some people are getting stuck in the form while waiting for a pending signature. To avoid this issue we want to let the user try to delete its case and start over. At first, we only want to try this feature whenever the case status type is `active:signature:pending`

## Explain your solution
- All code related to calculations has been moved to its own component
- Added a button in casesummary to be able to remove the current shown case
-
## How to test

1. Checkout this branch
2. Swap the application to run storybook
3. Fire upp the simulator by running the command `yarn ios` or `yarn android`
4. Make sure you have a case with status type `active:signature:pending`
5. Go to CaseSummary and the remove case buttons should be visible

## Tested environments

- [] Storybook on a iOS device/simulator.
